src/components/about-us/opening/index.js

```diff
@@ -0,0 +1,115 @@
+import React, { PureComponent } from 'react'
+import styled from 'styled-components'
+import { screen } from '../utils/screen'
+import Background from './background'
+import Header from './header'
+import CoverTitle from './cover-title'
+import MobileIntroContainer from './intro-container-mob'
+import startbutton from '../../../../static/asset/about-us/opening_start.png'
+import smoothScroll from 'smoothscroll'
+import { colors } from '../constants/styles'
+
+const containerWidth = {
+  mobile: '100%',
+  tablet: '719px',
+  desktop: '1024px',
+  overDesktop: '1440px'
+}
+
+const HeaderContainer = styled.div`
+  width: ${containerWidth.mobile};
+  background-color: transparent;
+  position: absolute;
+  top: 0;  
+  z-index: 2;
+`
+
+const Container = styled.div`
+  position: relative;
+  overflow: hidden;
+  background-color: ${colors.white};
+  width: ${containerWidth.mobile};
+  margin: 0 auto;
+  ${screen.mobile`
+    align-items: center;
+  `}
+  ${screen.desktop`
+    width: ${containerWidth.desktop};
+  `}
+  ${screen.overDesktop`
+    width: ${containerWidth.overDesktop};
+  `}  
+`
+
+const Cover = Container.extend`
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  height: 100vh;
+  padding: 80px;
+  ${screen.tablet`
+    display: block;
+    padding: 0;
+  `}
+  ${screen.mobile`
+    display: block;
+    height: 200vh;
+    padding: 0;
+  `}
+`
+
+const Startbutton = styled.div`
+  position: absolute;
+  left: 50%;
+  bottom: 0;
+  transform: translateX(-50%);
+  cursor: pointer;
+  ${screen.tabletBelow`
+    position: relative;
+    left: auto;
+    bottom: auto;
+    transform: none;
+    margin-top: 50px;
+  `}  
+`
+
+const DesktopStartbutton = Startbutton.extend`
+  ${screen.tabletBelow`
+    display: none;
+  `}     
+`
+
+export class Opening extends PureComponent {
+  constructor(props) {
+    super(props)
+    this._handleClick = this._handleClick.bind(this)
+  }
+  
+  _handleClick(event) {
+    event.preventDefault()
+    if (typeof window === 'undefined') return
```

Comment:
do we really need this condition?
if needed, please check `event.preventDefault` as well, since it is weird that `window` not exists but `event`exists.

Reply:
I will remove the condition examination.